client/components/tinymce/plugins/mentions/mentions.jsx

```diff
@@ -134,20 +136,43 @@ export class Mentions extends React.Component {
 		return null;
 	}
 
-	onKeyUp = ( { keyCode } ) => {
-		if ( includes( [ VK.ENTER, VK.SPACEBAR, VK.UP, VK.DOWN, 27 /* ESCAPE */ ], keyCode ) ) {
-			return this.hidePopover();
+	getSelectedSuggestionIndex() {
+		if ( ! this.matchingSuggestions ) {
+			return null;
 		}
 
-		const query = this.getQueryText();
+		if ( ! this.state.selectedSuggestionId ) {
+			return 0;
+		}
 
-		this.setState( {
-			showPopover: query !== null,
-			query,
-		} );
+		for ( let i = 0; i < this.matchingSuggestions.length; i++ ) {
+			if ( this.matchingSuggestions[ i ].ID === this.state.selectedSuggestionId ) {
+				return i;
+			}
+		}
+
+		return null;
```

Comment:
oops - this one here should be `findIndex( … )` while the next function should use `find( … )` - the difference being whether the `find` operation returns the index or the value at that index. otherwise the two are identical